processing-android

```diff
@@ -15,6 +15,11 @@
 
 # Thanks to Ferdinand Kasper for this build script. [fry]
 
+# Target version
+ANDROID_VERSION="17"
+
+# Your sketchbook location
+SKETCHBOOK_LOCATION="/path/to/your/sketchbook"
```

Comment:
@codeanticode I'll cook up an alternative version where this info is read from the preferences.txt, momento. I'll also look into creating build scripts for OSX + win32 (I might not know how to do those).